src/cmd/pay.rs

```diff
@@ -87,6 +87,8 @@ impl Cmd {
         let wallet = load_wallet(opts.files)?;
 
         let client = new_client(api_url(wallet.public_key.network));
+        
+        let base_url: String = api_url(wallet.public_key.network) + "/pending_transactions/";
```

Comment:
It'll compile into the same thing, but I would recommend calling api_url a single time and then cloning from there.

Something like:
```
let base_url = api_url(wallet.public_key.network);
let client = new_client(base_url.clone());
let pending_url = base_url + "/pending_transactions";
```

Reply:
Is leaving a space between the `wallet`, `client`, and `base_url` variables acceptable for readability?

Reply:
Yea there's no strong convention on when to put a single line break, IMO. So do as you like!